CONTRIBUTING.md

```diff
@@ -8,15 +8,26 @@ Please read through this document before submitting any issues or pull requests.
 
 ## Content
 
-- [❤️ Code of Conduct](#-code-of-conduct)
-- [Opening an issue](#opening-an-issue)
-- [How to work on an issue](#how-to-work-on-an-issue)
-- [How to make a pull request](#how-to-make-a-pull-request)
-  - [When is your pull request ready to be merged?](#-when-is-your-pull-request-ready-to-be-merged)
-- [How to merge a pull request](#how-to-merge-a-pull-request)
-- [Guideline commit messages](#guideline-commit-messages)
-  - [ℹ️ Semantic prefixes for commit messages](#-semantic-prefixes-for-commit-messages)
-  - [✍️ Writing a great commit message](#-writing-a-great-commit-message)
+- [Contributing Guidelines](#contributing-guidelines)
```

Comment:
@jlprat Sorry, I approved a little blindly. I just noticed that the`Content` section has new list items which are the main heading of the file and the "Content" heading itself. Was this intentional?

If it was not, I wonder if it is related to the similar situation I saw with my autoformatting in this PR https://github.com/aiven/klaw/pull/288, where all headings where added to the table of content, including the main heading and the "table of content" heading.

![Screenshot 2022-12-06 at 15 14 49](https://user-images.githubusercontent.com/20607294/206134679-9fcd448e-135d-4dd7-8663-30de066e8e70.png)

I ended up having to save without formatting to avoid the issue.